app/views/settings/edit.html.erb

```diff
@@ -3,7 +3,7 @@
 <div class="t-body">
   <h2 class="page__subheading"><%= t ".mfa.multifactor_auth" %></h2>
   <% if @user.mfa_enabled? %>
-    <h2><%= t(".mfa.level.title")%></h2>
+    <h2 class="mfa__header"><%= t(".mfa.level.title")%></h2>
     <p><%= t(".mfa.level_html") %></p>
```

Comment:
Since we're adding enhancements to this page, could you create an issue to revisit these translations?

Eg. 
<img width="771" alt="Screenshot 2023-07-17 at 10 36 16 AM" src="https://github.com/rubygems/rubygems.org/assets/42748004/4d030745-baf1-4f5a-a79d-10759a78a441">

```
You have enabled multi-factor authentication. Please click 'Update' to change your MFA level. Please refer to [RubyGems MFA guide](https://guides.rubygems.org/setting-up-multifactor-authentication) for more information on MFA levels.
```
`You have enabled multi-factor authentication. ` should be under the `MULTI-FACTOR AUTHENTICATION` heading, and I don't think `Please click 'Update' to change your MFA level.` is necessary at all.

and

<img width="769" alt="Screenshot 2023-07-17 at 10 34 41 AM" src="https://github.com/rubygems/rubygems.org/assets/42748004/b5fe105b-7c8c-4cc0-917f-6b087a39bbe4">

`Please refer to [RubyGems MFA guide](https://guides.rubygems.org/setting-up-multifactor-authentication) for more information on MFA levels.` doesn't fit quite right under OTP -> can we put this under `MULTI-FACTOR AUTHENTICATION` too?